.buildkite/pipeline.yml

```diff
@@ -0,0 +1,35 @@
+steps:
+  - label: ":eslint: Lint"
```

Comment:
shouldn't this be tslint now?

Reply:
I don't think we have a tslint emoji in buildkite :(